ynr/apps/elections/uk/templates/candidates/person-view.html

```diff
@@ -201,13 +201,11 @@ <h2>{% trans "Demographics:" %}</h2>
     </dl>
   {% endif %}
 
-  {% if extra_fields %}
+  {% if person.favourite_biscuit %}
```

Comment:
Do we want to show/hide this entire "Additional information" based on presence/absence of biscuit? Now that its a "core" field, should we just always show it regardless of whether its populated?

Reply:
Yeah, clearly this isn't the long term solution, but this does keep the existing functionality as biscuits are the only ExtraField. 

Happy to move it up the page to the person info if you think that's useful?

Reply:
Meh - if it makes more sense to think about it under the banner or redesigning the "person" page we can leave it for now and do it then.

Reply:
Yeah, I'll leave that for a later refactor of this page